vulture_os/toolkit/api_parser/infoblox_threat_defense/infoblox_threat_defense.py

```diff
@@ -0,0 +1,148 @@
+#!/home/vlt-os/env/bin/python
+"""This file is part of Vulture OS.
+
+Vulture OS is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+Vulture OS is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with Vulture OS.  If not, see http://www.gnu.org/licenses/.
+"""
+__author__ = "Nicolas Lançon"
+__credits__ = [""]
+__license__ = "GPLv3"
+__version__ = "4.0.0"
+__maintainer__ = "Vulture OS"
+__email__ = "contact@vultureproject.org"
+__doc__ = 'INFOBLOX THREAT DEFENSE API'
+__parser__ = 'INFOBLOX_THREAT_DEFENSE'
+
+from json import dumps as json_dumps
+import logging
+
+from requests import Session as requests_session
+
+from datetime import timedelta
+from django.utils import timezone
+from django.conf import settings
+from toolkit.api_parser.api_parser import ApiParser
+
+logging.config.dictConfig(settings.LOG_SETTINGS)
+logger = logging.getLogger('api_parser')
+
+class InfobloxThreatDefenseAPIError(Exception):
+    pass
+
+class InfobloxThreatDefenseParser(ApiParser):
+    HEADERS = {
+        "Content-Type": "application/json",
+        'Accept': 'application/json'
+    }
+
+    def __init__(self, data):
+        super().__init__(data)
+
+        self.infoblox_threat_defense_host = data["infoblox_threat_defense_host"].rstrip("/")
+        if not self.infoblox_threat_defense_host.startswith('https://'):
+            self.infoblox_threat_defense_host = f"https://{self.infoblox_threat_defense_host}"
```

Comment:
what if the value starts with `http://`?
prefered method:

```suggestion
        self.infoblox_threat_defense_host = "https://" + data["infoblox_threat_defense_host"].rstrip("/").split("://")[-1]
```